docs/guides/visualization.rst

```diff
@@ -0,0 +1,287 @@
+    tiled_layer = raster_layer.tile_to_layout(layout=gps.GlobalLayout(), target_crs=3857)
+
+
+.. _visualizing:
+
+Visualizing Data in GeoPySpark
+==============================
+
+Data is visualized in GeoPySpark by running a server which allows it to
+be viewed in an interactive way. Before putting the data on the server,
+however, it must first be formatted and colored. This guide seeks to go
+over the steps needed to create a visualization server in GeoPySpark.
+
+.. _pyramid:
+
+Pyramid
+-------
+
+The ``pyramid`` class represents a list of ``TiledRasterLayer``\ s that
+represent the same area where each layer is a level within the pyramid
+at a specific zoom level. Thus, as one moves up the pyramid (starting a
+level 0), the image will become more zoomed by a power of 2 for each
+level. It is this varying level of detail that allows an interactive
+tile server to be created from a ``Pyramid``. This class is needed in
+order to create visualizations of the contents within its layers.
+
+Creating a Pyramid
+~~~~~~~~~~~~~~~~~~
+
+There are currently two different ways to create a ``Pyramid`` instance:
+Through the ``TiledRasterLayer.pyramid`` method or by constructing it by
+passing in a ``[TiledRasterLayer]`` or
+``{zoom_level: TiledRasterLayer}`` to ``Pyramid``.
+
+Any ``TiledRasterLayer`` with a ``max_zoom`` can be pyramided. However,
+the resulting ``Pyramid`` may have limited functionality depending on
+the layout of the source ``TiledRasterLayer``. In order to be used for
+visualization, the ``Pyramid`` **must** have been created from
+``TiledRasterLayer`` that was tiled using a ``GlobalLayout`` and whose
+tile sizes are a power of 2.
+
+Via the pyramid Method
+^^^^^^^^^^^^^^^^^^^^^^
+
+When using the ``pyramid`` method, a ``Pyramid`` instance will be
+created with levels from 0 to ``TiledRasterlayer.zoom_level``. Thus, if
+a ``TiledRasterLayer`` has a ``zoom_level`` of 12 then the resulting
+``Pyramid`` will have 13 levels that each correspond to a zoom from 0 to
+12.
+
+.. code:: python3
+
+    pyramided = tiled_layer.pyramid()
+    pyramided
+
+Contrusting a Pyramid Manually
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code:: python3
+
+    gps.Pyramid([tiled_layer.tile_to_layout(gps.GlobalLayout(zoom=x)) for x in range(0, 13)])
+
+.. code:: python3
+
+    gps.Pyramid({x: tiled_layer.tile_to_layout(gps.GlobalLayout(zoom=x)) for x in range(0, 13)})
+
+Computing the Histogram of a Pyramid
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+One can produce a ``Histogram`` instance representing all of the layers
+within a ``Pyramid`` via the ``get_histogram`` method.
+
+.. code:: python3
+
+    hist = pyramided.get_histogram()
+    hist
+
+RDD Methods
+~~~~~~~~~~~
+
+``Pyramid`` contains methods for working with the ``RDD``\ s contained
+within its ``TiledRasterLayer``\ s. A list of these can be found
+here :ref:`rdd-methods`. When used, all internal ``RDD``\ s
+will be operated on.
```

Comment:
This change would be out of scope for this PR but would like to point out that we're making a transition from talking about these things are `RDD`s, which they're deep down, to `Layer` which they are conceptually. This is what prompted renaming from `TiledRasterRDD` to `TiledRasterLayer` and in general we should avoid leaking the implementation details into docs unless we're specifically talking about them or their consequences (performance, error messages, execution environment)

Reply:
That's a good point. I do agree that is outside of the scope of this PR, though. I'm going to create an issue that addresses this and other problems with the docs now.